Assets/Scenes/Scripts/RandomRotator.cs

```diff
@@ -1,18 +1,23 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Assertions;
 
-public class RandomRotator : MonoBehaviour {
-	public float tumble;
+[System.Serializable]
+public class RandomRotator : MonoBehaviour 
+{
+	[SerializeField]
+	private float tumble;
 
-	// Use this for initialization
-	void Start () {
+	void Awake()
+	{
+		Assert.IsTrue(tumble == 2);
```

Comment:
use, `Assert.AreEqual(expected, actual);`. Also if it needs to be equal to 2 then just set it to 2 and don't allow the user the ability to change it.

```
private const float tumble = 2f;
```